cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/monitor/RuntimeClient.java

```diff
@@ -115,6 +123,16 @@ public void sendMessages(ProgramRunId programRunId,
       }
 
       throwIfError(programRunId, urlConn);
+      // fetch grace period for stopping
```

Comment:
Should we have this as part of a future PR?

Reply:
should it be part of the response where stopping status is sent from runtime handler? That way we dont need to wait for 2 calls for getting all the details

Reply:
As discussed offline, keeping the below lines. Also, the stopping status will be part of the response from runtime handler